src/test/java/igym/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,66 @@
+package igym.controllers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+
+import igym.entities.UserEntity;
+import igym.repositories.UserRepository;
+
+@SpringBootTest
+@ActiveProfiles("test")
+public class UserControllerTest {
+
```

Comment:
```suggestion
        UserEntity user1 = new UserEntity("Maria Clown");
        UserEntity user2 = new UserEntity("Johan Marry");
```
I would suggest a simple table test ( all resources that your test needs to run) instead declaring user in each test case.